### ISSUE TITLE
Flashing purple screen when using gpu-api=vulkan

### ISSUE THREAD

**Issue by PuraIgnis** [mpv-player/mpv#14026]
### Important Information

Provide following Information:
- mpv version ( **latest version by shinchiro** )
- Windows Version ( **Windows 10** )
- Source of the mpv binary - ( **https://sourceforge.net/projects/mpv-player-windows/files/64bit-v3/** )
- If known which version of mpv introduced the problem - ( **I am using the latest version** )
- GPU model, driver and version - ( **AMD Ryzen 4800H with Radeon Graphics. Driver version: 31.0.21912.14** )
- Possible screenshot or video of visual glitches 
![purple flashing screen](https://github.com/mpv-player/mpv/assets/120215268/aa9fc0fb-437e-4b72-879e-a551a308bbd3)

### Reproduction steps
It happens when I use --gpu-api=vulkan. Changing it to d3d11 would eliminate the issue. 

### Expected behavior
Video plays.

### Actual behavior
Flashing screen, but audio is still there.

### Log file
([output.txt](https://github.com/mpv-player/mpv/files/15159911/output.txt)))

**Comment by WilliamK7** [comment]
Hi, I'm having the same issue with `gpu-context = winvk`
Here is the log file
[mpv-lazy.log](https://github.com/user-attachments/files/16670636/mpv-lazy.log)

**Comment by WilliamK7** [comment]
I updated my NVIDIA drive from 551.23 to 560.94, and the issue disappeared.

**Comment by WilliamK7** [comment]
Sorry, but the issue reappears.
Here is the new log file
[mpv-lazy.log](https://github.com/user-attachments/files/16737013/mpv-lazy.log)

**Comment by WilliamK7** [comment]
It's weird that after I restart my computer, the issue disappears again, so this issue is not related to the Nvidia drive problem.

**Comment by WilliamK7** [comment]
> Also enabling resizable bar might help to mitigate the issue.

That's right, I tried to enable it in my BIOS settings several days ago. After that, the problem hasn't occurred so far.

**Comment by Arxhexa** [comment]
Can confirm this problem still exists with latest build on Windows

Got `[vo/gpu-next/libplacebo] Allocation of size  252K failed: VK_ERROR_OUT_OF_DEVICE_MEMORY!`

with RAM of 64GB and VRAM of 22GB
```
vo=gpu-next
gpu-context=winvk
gpu-api=vulkan
hwdec=no
```

**Comment by Arxhexa** [comment]
I might have figured out one possible reason why this problem happens after using my PC for a while, and why restarting it temporarily solves the issue. I have an NVIDIA graphics card with RTX Video Super Resolution turned on. Whenever I open Chrome and play a video that's below 4K resolution, it kicks in. After that I will get a purple screen when using Vulkan api in MPV. Enabling RTX Video HDR doesn't cause this problem.

But the original post was on an AMD platform, so this might not be an NVIDIA driver issue?

**Comment by Arxhexa** [comment]
Restarting video driver using `win+ctrl+shift+b` cannot fix the problem if the purple screen has already happened, a full system restart is required.

**Comment by pufss** [comment]
> I might have figured out one possible reason why this problem happens after using my PC for a while, and why restarting it temporarily solves the issue. I have an NVIDIA graphics card with RTX Video Super Resolution turned on. Whenever I open Chrome and play a video that's below 4K resolution, it kicks in. After that I will get a purple screen when using Vulkan api in MPV. Enabling RTX Video HDR doesn't cause this problem.
> 
> But the original post was on an AMD platform, so this might not be an NVIDIA driver issue?

Seems to be the fix for me. I disabled super resolution in the nvidia control panel and then closed my web browser / anything else that might be using super resolution. Re-opened MPV and it was back to playing my video normally with no flashing purple screen.

**Comment by sergeyganago** [comment]
> disabled super resolution in the nvidia control panel and then closed my web browser

works for me too
